shell/client/_referrals.scss

```diff
@@ -0,0 +1,91 @@
+.referrals {
+  @extend %padded-info-page;
+
+  .pageheader {
+    background-color: #6d5eb7;
+    color: white;
+
+    // Use DIV nesting so that we can have two different backgrounds.
+    .leftbackground {
+      background-image: url("/referral-graphics/referral-edge.svg");
```

Comment:
Just a comment, not a problem:

I created a new directory `shell/public/referral-graphics/` and put them all in there.